src/test/java/org/sagebionetworks/bridge/sdk/integration/StudyConsentTest.java

```diff
@@ -6,80 +6,91 @@
 
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.sagebionetworks.bridge.sdk.ResearcherClient;
+
+import org.sagebionetworks.bridge.Tests;
+import org.sagebionetworks.bridge.sdk.DeveloperClient;
 import org.sagebionetworks.bridge.sdk.Roles;
 import org.sagebionetworks.bridge.sdk.TestUserHelper;
 import org.sagebionetworks.bridge.sdk.TestUserHelper.TestUser;
 import org.sagebionetworks.bridge.sdk.exceptions.BridgeSDKException;
 import org.sagebionetworks.bridge.sdk.models.ResourceList;
+import org.sagebionetworks.bridge.sdk.models.holders.GuidVersionHolder;
 import org.sagebionetworks.bridge.sdk.models.subpopulations.StudyConsent;
+import org.sagebionetworks.bridge.sdk.models.subpopulations.Subpopulation;
+import org.sagebionetworks.bridge.sdk.models.subpopulations.SubpopulationGuid;
 
 public class StudyConsentTest {
 
-    private TestUser researcher;
+    private TestUser admin;
+    private TestUser developer;
+    private SubpopulationGuid subpopGuid;
 
     @Before
     public void before() {
-        researcher = TestUserHelper.createAndSignInUser(StudyConsentTest.class, true, Roles.RESEARCHER);
+        admin = TestUserHelper.getSignedInAdmin();
+        developer = TestUserHelper.createAndSignInUser(StudyConsentTest.class, true, Roles.DEVELOPER);
     }
 
     @After
     public void after() {
-        researcher.signOutAndDeleteUser();
+        if (subpopGuid != null) {
+            admin.getSession().getAdminClient().deleteSubpopulationPermanently(subpopGuid);    
+        }
+        developer.signOutAndDeleteUser();
     }
 
     @Test(expected=BridgeSDKException.class)
-    public void mustBeResearcherToAdd() {
+    public void mustBeDeveloperToAdd() {
```

Comment:
The name of the test suggests that non-developers should be unable to add consent, but we don't test for that. Maybe rename this test, and add test to verify researcher can't change consent (this might be complicated, do we need to prevent users of the old SDK from modifying consent using researcher client?)